static/resources/js/members-common.js

```diff
@@ -87,13 +87,14 @@ function addMember(members_table) {
         }
     }
 
-    if($('#first_name').val()=="" || $('#last_name').val()=="" || $('#date_of_birth').val()=="" ||
-         $('select[name=church-role-select]').val()==null) {
-
+    if($('#first_name').val()=="" || $('#last_name').val()=="" || $('select[name=church-role-select]').val()==null){
         $('#required-fields-alert').show();
 
     }
+    else if(dateFormatValidator($('#date_of_birth').val())== "False"){
```

Comment:
is this the only place where we want to check the date? What about in the edit member function? Look to see if there are other places around the code where we work with dates where we might need to do similar checks and add this logic there too. :)

Reply:
Check for baptismal date and membership date as well!